.github/workflows/deploy.yml

```diff
@@ -52,35 +52,35 @@ jobs:
     runs-on: macos-latest
 
     steps:
-    - uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
 
-    - name: Setup Node
-      uses: actions/setup-node@v2
+      - name: Setup Node
+        uses: actions/setup-node@v2
 
-    - name: Install Bundler
-      run: gem install bundler
+      - name: Install Bundler
+        run: gem install bundler
 
-    - name: Decrypt Environment Variables and Apple API Key
-      run: (cd .. && yarn secrets -d ${{ secrets.ENCRYPTION_PASSWORD }})
+      - name: Decrypt Environment Variables and Apple API Key
+        run: (cd .. && npx @apollosproject/apollos-cli secrets -d ${{ secrets.ENCRYPTION_PASSWORD }})
```

Comment:
Small tweak here :) I believe you can remove the `cd ..` since the script needs to be run alongside the .env files.

Reply:
Good catch! I can make that change, but the `cd ..` was there before — we're confident that's safe to update?

Reply:
Yep - since the behavior of the script has changed. you used to need to run it from the root

Reply:
Updated that, and removed all the whitespace changes my IDE decided were important :).